src/main/java/eggventory/model/items/Stock.java

```diff
@@ -173,47 +179,49 @@ public String toString() {
-        System.out.println(description + ": " + numAvailable() + " available. " + loaned + " on loan. "
-                + lost + " lost. (" + quantity + " total.)");
-    }
-
-    /**
-     * Prints the name and number of available items. Used as part of printing a list of available items.
-     * Format example: 560ohm Resistors: 280
-     * To be used with the 'stock' command.
-     */
-    public void printAvailable() {
-        System.out.println(description + ": " + numAvailable() + " available.");
-    }
-
-    /**
-     * Prints the name and number of items on loan. Used as part of printing a list of items on loan.
-     * Format example: 560ohm Resistors: 100
-     * To be used with the 'stock loan' command.
-     */
-    public void printLoan() {
-        System.out.println(description + ": " + loaned + " on loan.");
-    }
-
-    /**
-     * Prints the name and number of lost items. Used as part of printing a list of lost items.
-     * Format example: 560ohm Resistors: 20
-     * To be used with the 'stock lost' command.
-     */
-    public void printLost() {
-        System.out.println(description + ": " + lost + " lost.");
-    }
+        return stockType + "," + stockCode + "," + quantity + "," + description + "," + minimum;
+    }
+
+    //    //TODO: Fix methods below for new UI.print() implementation.
+    //
+    //    /**
+    //     * Prints the complete details of all the items of this type.
+    //     * Format example: 560ohm Resistors: 280 available. 100 on loan. 20 lost. (400 total.)
+    //     * To be used with the 'stock all' command.
+    //     */
+    //    public String printAll() {
+    //        String output = description + ": " + numAvailable() + " available. " + loaned + " on loan. "
+    //                + lost + " lost. (" + quantity + " total.)";
+    //
+    //
+    //    }
+    //
+    //    /**
+    //     * Prints the name and number of available items. Used as part of printing a list of available items.
+    //     * Format example: 560ohm Resistors: 280
+    //     * To be used with the 'stock' command.
+    //     */
+    //    public void printAvailable() {
+    //        System.out.println(description + ": " + numAvailable() + " available.");
+    //    }
+    //
+    //    /**
+    //     * Prints the name and number of items on loan. Used as part of printing a list of items on loan.
+    //     * Format example: 560ohm Resistors: 100
+    //     * To be used with the 'stock loan' command.
+    //     */
+    //    public void printLoan() {
+    //        System.out.println(description + ": " + loaned + " on loan.");
+    //    }
+    //
+    //    /**
+    //     * Prints the name and number of lost items. Used as part of printing a list of lost items.
+    //     * Format example: 560ohm Resistors: 20
+    //     * To be used with the 'stock lost' command.
+    //     */
+    //    public void printLost() {
+    //        System.out.println(description + ": " + lost + " lost.");
+    //    }
```

Comment:
man i should reclaim these as unused code i guess

Reply:
We need to rewrite these if we still want them. Cause they are obsolete lardy atm

Reply:
after we implement lost and loan properly... 1.4 
i'll move these at some point